MeesGame/Gameplay/GameObjects/Tile.cs

```diff
@@ -54,14 +56,27 @@ public override bool CanPlayerMoveHere(Player player)
 
         public override bool IsActionForbiddenFromHere(Player player, PlayerAction action)
         {
-            //special actions are not allowed on floor tiles, because there is no special action available.
-            return action == PlayerAction.SPECIAL;
+            //A player can move into all directions from a floor tile.
+            //Special actions are allowed on floor tiles, if there is a tile which allows a special action next to it.
+            if (IsNextToSpecialTile())
+            {
+                return false;
+            }
+            //Special actions are not allowed on a floor tile, if it is surrounded by tiles that don't allow special actions. 
+            else
+                return action == PlayerAction.SPECIAL;
+        }
+
+        //TODO
+        public bool IsNextToSpecialTile()
+        {
+            return false;
         }
     }
 
     class WallTile : Tile
     {
-        public WallTile(int layer = 0, string id = "") : base("wall", TileType.Wall, layer, id)
+        public WallTile(string assetName = "fourWayWallTile", TileType tt = TileType.Wall, int layer = 0, string id = "") : base(assetName, tt, layer, id)
```

Comment:
Using "wall" crashes the game, so I think it's good to have a default that doesn't.